src/events/order/cd-event-ticket.vue

```diff
@@ -70,6 +73,23 @@
             }),
         }));
       },
+      userType() {
+        if (this.isNinja) {
+          return 'ninja';
```

Comment:
`youth` instead of `ninja`. Using ninja in our code is fine, but we decided a good while ago to go with `youth` over `ninja` for user-facing text going forward.

Reply:
gotcha